tests/manifold/projection_manifold_01.cc

```diff
@@ -0,0 +1,74 @@
+//-------------------------------------------------------------------
+//    Copyright (C) 2016 by the deal.II authors.
```

Comment:
sure, i am not speaking about that. The content is different to what i quoted above (unless there are some glitches in GitHub which does not show the current version).